src/Microsoft.Diagnostics.Tools.RuntimeClient/Eventing/EventPipeClient.cs

```diff
@@ -48,7 +48,7 @@ public static IEnumerable<int> ListAvailablePorts()
         public static Stream CollectTracing(int processId, SessionConfiguration configuration, out ulong sessionId)
         {
             sessionId = 0;
-            var message = new IpcMessage(DiagnosticsServerCommandSet.EventPipe, (byte)EventPipeCommandId.CollectTracing, configuration.Serialize());
+            var message = new IpcMessage(DiagnosticsServerCommandSet.EventPipe, (byte)EventPipeCommandId.CollectTracing2, configuration.Serialize());
```

Comment:
Thinking about this a little more, I think we should also make the tools compatible with the preview 7 bits. Although we could probably get away with not supporting it there are two advantages:
a) Every time we break compat during our previews we make it harder for customers to get matched bits
b) In the future we'll need to be able to reasonably support older versions of the runtime. If the versioning workflow doesn't work reasonably we need to get that solved.

My suggestion in the RuntimeClient is to expose both CollectTracing and CollectTracing2. In dotnet-counters you can try CollectTracing2 first, and if that gives a not supported error fallback to CollectTracing. Dotnet-trace you could leave using CollectTracing for now to lower the amount of changes and the cases that need to be tested.